.github/workflows/bazel.yml

```
@@ -0,0 +1,14 @@
+name: Bazel
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: ['*']
+
+jobs:
+  MacOS:
+    runs-on: macos-11
+    steps:
+      - uses: actions/checkout@v2
+      - run: bazelisk test //Tests:UnitTests --test_env=PROJECT_ROOT=$(bazelisk info workspace) --spawn_strategy=standalone
```

Let's also build the CLI here

```suggestion
      - run: bazelisk test //Tests:UnitTests --test_env=PROJECT_ROOT=$(bazelisk info workspace) --spawn_strategy=standalone
      - run: bazelisk build sourcekitten
```

It's currently failing because the checksum wasn't updated for ArgumentParser, but CI is passing because we're not checking this.